### PR TITLE
[ci skip] Fix Grammatical Errors and Eliminate Ambiguous Wordings in Multiple Database Documents

### PR DESCRIPTION
**Pull request by aeroastro**
### Summary

While reading the Rails Guides, I noticed there are some rooms for improvement
for guides on Multiple Databases.

* Add comma after the introductory clause.
* Add backtick for the symbols used in the program.
  * `primary`
  * `false`
* Add descriptive information on ambiguous words.
  * users => database users
  * use the first configuration => use the first configuration as default
  * a recent write => a recent write by the requesting user
  * for an environment => for each environment
  * both a `role` and `shard` => both a `role` and a `shard`
* Add missing period at the end of a sentence.

and misc improvement that I think helps readers to understand the Rails more.

### CODE REVIEW

File: guides/source/active_record_multiple_databases.md [hunk 1]
```diff
@@ -222,13 +222,10 @@ class Dog < AnimalsRecord
 end
 ```
 
-Note: Since Rails doesn't know which database is the replica for your writer you will need to
-add this to the abstract class after you're done.
-
```
Comment (aeroastro) [inline guides/source/active_record_multiple_databases.md:227]: Sorry, but I noticed that this is not fully addressed at the previous document.
So, I will revert this removal and squash the commit. (Maybe we can find the best place afterwards)

**Comment by aeroastro** [comment]
I have fixed the title because this PR is rather fixing grammatical errors (punctuations) and ambiguous wording than just an enhancement.

File: guides/source/active_record_multiple_databases.md [hunk 2]
```diff
@@ -57,10 +57,10 @@ Let's add a replica for the first configuration, and a second database called an
 replica for that as well. To do this we need to change our `database.yml` from a 2-tier
 to a 3-tier config.
 
-If a primary configuration is provided this will be used as the "default" configuration. If
-there is no configuration named "primary" Rails will use the first configuration for an
-environment. The default configurations will use the default Rails filenames. For example
-primary configurations will use `schema.rb` for the schema file whereas all other entries
+If a primary configuration is provided, it will be used as the "default" configuration. If
+there is no configuration named `primary`, Rails will use the first configuration as default
```
Comment (rafaelfranca) [inline guides/source/active_record_multiple_databases.md:61]:
```suggestion
there is no configuration named `"primary"`, Rails will use the first configuration as default
```

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

**Comment by aeroastro** [comment]
Thank you for reviewing this 🐱 
I have committed the suggestion.

**Pull request merged**